frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -223,7 +222,6 @@ const Map: React.FC<MapProps> = ({
   const closeMarkerPopup = (e: any) => {
     if (e.target._animateToZoom === mapRef.current?.leafletElement.getZoom()) {
       setLayerPopup(undefined);
-      dispatch(storeParcelDetail(null));
```

Comment:
Can you confirm that this doesn't adversely affect the 'zoom' functionality please?